Pod/Classes/NSManagedObject+VOKManagedObjectAdditions.h

```diff
@@ -45,14 +45,14 @@ typedef void(^VOKManagedObjectsReturnBlock)(VOKArrayOfManagedObjects *managedObj
 + (NSString *)vok_entityName;
 
 /**
- A convenience methods to create a new instance of a NSManagedObject subclass.
+ A convenience method to create a new instance of a NSManagedObject subclass.
  @return    A new managed object subclass in the main context.
  */
 + (instancetype)vok_newInstance;
 
 /**
- A convenience methods to create a new instance of a NSManagedObject subclass.
- @param contextOrNil    The managed object context to insert the new object.  If nil, the main context will be used.
+ A convenience method to create a new instance of a NSManagedObject subclass.
+ @param contextOrNil    The managed object context in which to insert the new object.  If nil, the main context will be used.
  @return                A new managed object subclass in the main context.
  */
 + (instancetype)vok_newInstanceWithContext:(nullable NSManagedObjectContext *)contextOrNil;
```

Comment:
At some point, we might want to standardize the preposition we use to indicate the managed object context.  "with context", "for context", and "in context" being the three options that I think I've seen.